compiler/src/dmd/errors.d

```diff
@@ -105,6 +105,7 @@ class ErrorSinkCompiler : ErrorSink
 /**
  * Color highlighting to classify messages
  */
+alias Color = dmd.console.Color;
```

Comment:
???

Reply:
This problem baffles me:
```
src\dmd\dtemplate.d(7910): Error: function `dmd.dtemplate.TemplateMixin.accept` does not override any function
src\dmd\dtoh.d(309):        called from here: `generateMembers()`
```
which pops up here and is the same issue I found with the Color thing above works around. Something about `generateMembers()`

Reply:
If I take out line 108,
```
src/dmd/errors.d(111): Error: undefined identifier `Color`
src/dmd/dtoh.d(309):        called from here: `generateMembers()`
```

Reply:
Hmm, I think forward references due to cyclical imports is causing this problem.